src/varpubs/pubmed_db.py

```diff
@@ -14,24 +22,170 @@ class PubmedArticle(SQLModel, table=True):
     doi: str
 
 
+class TermToPMID(SQLModel, table=True):
+    term: str = Field(nullable=False)
+    pmid: int = Field(Integer, nullable=False)
+    gene: str = Field(nullable=False)
+
+    __table_args__ = (sqlalchemy.PrimaryKeyConstraint("term", "pmid", "gene"),)
+
+
 @dataclass
 class PubmedDB:
     path: Path
-    _engine: sqlalchemy.engine.base.Engine | None = field(init=False)
+    vcf_paths: Iterable[Path]
+    email: str
+    _engine: Optional[sqlalchemy.engine.base.Engine] = field(init=False, default=None)
 
     def deploy(self) -> None:
-        # wipe existing database
-        if self.path.exists():
-            self.path.unlink()
-        # create parent directories if they don't exist
-        if self.path.parent != Path():
-            self.path.parent.mkdir(parents=True, exist_ok=True)
-        # create the database
+        logger.info(f"Deploying database at: {self.path}")
+        logger.info(f"Input VCF paths: {[str(p) for p in self.vcf_paths]}")
+        logger.info(f"Entrez email: {self.email}")
+
+        self.create_tables()
+        terms = self.extract_terms()
+        Entrez.email = self.email
+
+        with Session(self.engine) as session:
+            all_pmids = set()
+            term_to_pmids = {}
+
+            for term in terms:
+                try:
+                    handle = Entrez.esearch(db="pubmed", term=term, retmax=3)
```

Comment:
Can't we put all terms into a big OR?
Like
```python
query = " OR ".join(terms)
```

This way, we would only have a single query.

Reply:
The matching between individual terms and PMIDs can then be done afterwards, on our database entries.